package.json

```diff
@@ -11,6 +11,8 @@
     "bip39": "^3.0.2",
     "crypto-js": "^4.0.0",
     "lodash": "^4.17.19",
+    "moment": "^2.27.0",
```

Comment:
Are `moment` and `fetch` being used?

Reply:
Good catch. I was using them locally because I've got some uncommitted webflow testing scripts in this repo. Removed in https://github.com/Kava-Labs/javascript-sdk/pull/41/commits/0f1bef1b1854553c7b7a0eee897ecdd07d8ac60a.